spark-indexer/src/main/resources/default-proteome-synonyms.txt

```diff
@@ -9,6 +9,7 @@
 10090	Mouse,Laboratory mouse,House mouse	high
 10116	Rat,Brown rat,Laboratory rat,Norway rat,Buffalo rat,R. norvegicus	high
 559292	Baker’s yeast,Brewer’s yeast,S. cerevisiae	high
+4932	Baker’s yeast,Brewer’s yeast,S. cerevisiae	medium
```

Comment:
Should we add 284812 and 4896 for proteome as well?  https://www.uniprot.org/proteomes?query=organism_id%3A284812,